### PR TITLE
Display private|admin lock sign on account-related tabs.

### PR DESCRIPTION
**Pull request by isoos**
image

(first underline was my mouse hover over the menu item, while the last is the active tab).

### CODE REVIEW

File: app/lib/frontend/templates/admin.dart [hunk 1]
```diff
@@ -64,7 +64,7 @@ String renderAccountPackagesPage({
     headerHtml: _accountDetailHeader(user, userSessionData),
     tabs: [
       Tab.withContent(
-          id: 'packages', title: 'My packages', contentHtml: tabContent),
+          id: 'my-packages', title: 'My packages', contentHtml: tabContent),
```
Comment (sigurdm) [inline app/lib/frontend/templates/admin.dart:67]: Will this change break existing links to that tab?

Comment (sigurdm) [reply app/lib/frontend/templates/admin.dart:67]: Perhaps just let the Tab.withContent take another parameter telling if it's private?

Comment (isoos) [reply app/lib/frontend/templates/admin.dart:67]: > Will this change break existing links to that tab?

No, tabs are no longer referenced by their ids, and these are private (authenticated) URLs anyway.

**Review by sigurdm — APPROVED** [review]
LGTM

**Pull request merged**